packages/components/src/components/page-error-container/page-error-container.tsx

```diff
@@ -16,6 +17,10 @@ const PageErrorContainer = ({ error_header, error_messages, ...props }: TPageErr
     if (error_header && error_messages) {
         return <PageError header={error_header} messages={error_messages} {...props} />;
     }
+    // If there are error messages from the backend, show an error modal with the messages
+    if (error_messages) {
+        return <ErrorModal messages={error_messages} />;
+    }
```

Comment:
This might affect the errors all over the code and I can see that there are many places that `error_header` might be null, can you add proper QA checklist and let them know to test the app errors thoroughly!

Reply:
The main reason for this `error_header` null was to show the Error Message from the backend. Still, I updated the QA checklist.